src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -75,11 +84,32 @@ const DeletePanel: FC = () => {
                 testID="delete-org--button"
                 text="Delete"
                 icon={IconFont.Trash_New}
-                onClick={handleDeleteClick}
+                onClick={handleDeleteFreeAccount}
               />
             </FlexBox.Child>
           </>
         )}
+        {CLOUD &&
+          isFlagEnabled('createDeleteOrgs') &&
+          (account.type === 'pay_as_you_go' || account.type === 'contract') && (
```

Comment:
might be worth abstracting this expression out into a variable

Reply:
👍